### DISCUSSION TITLE
Rule to ban all statements

### DISCUSSION THREAD

**Discussion by FbN** [eslint/eslint#16166] [Q&A] [answered]
Hi people,

Is possible to write a rule to ban any statements? And permit only expressions?

So banning: code blocks, if, switch, throw, try, catch, for....

So all statements except: import, export, const, exit

**Comment by nzakas** [comment, marked as answer]
Yes. You just need to decide on the complete list of nodes you want to block. Use astexplorer.net to figure that out.
